data/mt/retired/Dan-Bartel-5d98dd7f-fa7f-4f57-a5f3-d65181feee04.yml

```diff
@@ -7,7 +7,8 @@ email: danbartel2@gmail.com
 party:
 - name: Republican
 roles:
-- type: lower
```

Comment:
@bfossen-ce this person is in the pull twice. They served in the house of reps until November 8, 2021 at which point he was appointed to the state senate